apps/frontend/src/libs/components/sidebar/sidebar-item/styles.module.css

```diff
@@ -0,0 +1,28 @@
+.sidebar-item > * {
```

Comment:
Don't use * for selectors, we don't need it here. The layout supposes that we apply styles only to the needed element, not all inside elements inside item. And the same for the selectors below.